inji-web/src/locales/fr.json

```diff
@@ -10,27 +10,25 @@
   },
   "HomePage": {
     "Intro": {
-      "title": "Le téléchargement d'un identifiant se fait en un seul clic!",
+      "title": "Le téléchargement d'un identifiant se fait en un seul clic !",
       "subTitle": "Veuillez rechercher l'émetteur et, à l'étape suivante, sélectionnez les informations d'identification à télécharger.",
       "searchText": "Rechercher des émetteurs par nom"
     },
     "emptyContainerContent": "Aucun émetteur trouvé. ",
     "containerHeading": "Liste des émetteurs",
-    "containerSubHeading": "Recherchez votre émetteur de confiance et choisissez un type d'identifiant à l'étape suivante.",
```

Comment:
Why this section is removed, please do check this in all other locale files as well